### ISSUE TITLE
Where to download the dataset?

### ISSUE THREAD

**Issue by BugMaker2002** [yuhang1070/rPPG_Strong_Baseline#3]
Where to download the dataset? I found a lot of information on the Internet but the owner of UBFC-rPPG told me the dataset was available on request

**Comment by alaskax8** [comment]
Yes you can request the dataset from the owner. Just send them a message and they will provide you with a download link. But this unfortunately will not help you to run the code in the repository as there is a lot missing actually.

**Comment by BugMaker2002** [comment]
> Yes you can request the dataset from the owner. Just send them a message and they will provide you with a download link. But this unfortunately not help you to run the in the repository as there is a lot missing actually.

thank you very much
